saleor/core/forms.py

```diff
@@ -0,0 +1,33 @@
+from django import forms
+
+
+class AjaxSelect2ChoiceField(forms.ChoiceField):
```

Comment:
Since this is our custom and reusable widget, we should add docstrings here and in our custom methods (not belonging to the Django's forms API)